config/config.go

```diff
@@ -499,6 +507,7 @@ func NewConfig() (c *Config) {
 		IstioLabels: IstioLabels{
 			AppLabelName:       "app",
 			InjectionLabelName: "istio-injection",
+			InjectionLabelRev:  "istio.io/rev",
```

Comment:
I do not see this in the operator PR: https://github.com/kiali/kiali-operator/pull/347/files

Should this be in the Kiali CR or is this not configurable by the user?

Reply:
added